packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelFormFields/DefaultValueField.tsx

```diff
@@ -163,7 +163,7 @@ export default function DefaultValueField<T>(props: DefaultValueFieldProps<T>) {
     <TextField
       {...TextFieldProps}
       type={'text'}
-      value={value}
+      value={value || undefined}
```

Comment:
Fix a case where warnings were logged because `value` may have been `null`, which is not a valid input value. Now if `value` is `null`, the prop becomes `undefined`, which is valid